members/templates/members/detail.html

```diff
@@ -3,130 +3,16 @@
 {% load member_lending_table %}
 {% block title %}{% if member.is_blacklisted %}<p style="color:red;">BLACKLISTED</p>{%endif %}Member: {{ member.name }}{% endblock %}
 {% block content %}
+    <style>
+     .table{
+         font-size: 12px;
+     }
+</style>
+    <div class="container-fluid" style="width:100%">
+  <div class="row justify-content-md-center">
     {{ extra }}<br/>
```

Comment:
Extra now shoves the cards around.
![afbeelding](https://github.com/user-attachments/assets/404cb98f-ceb0-481f-8e9e-432d1613d817)